magmartfront/src/APIs/getUserById.ts

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+import { UserData } from '@/components/myTypes/UserTypes';
+
+async function GetUserById( id: string ){
+    return Promise.resolve(axios.get<UserData>(`http://localhost:3001/users/${id}`))
```

Comment:
Padronizar a chamada usando variável de ambiente.
```suggestion
    return Promise.resolve(axios.get<UserData>(`${process.env.API_URL}/users/${id}`))
```